app/src/org/commcare/activities/CommCareWiFiDirectActivity.java

```diff
@@ -808,11 +808,11 @@ private void updateStatusText() {
     }
 
     public static void copyFile(InputStream inputStream, OutputStream out) throws IOException {
-        Logger.log(TAG, "File server copying file");
         if (inputStream == null) {
             Logger.log(TAG, "Input stream null");
             throw new IOException("Got null input stream");
         }
+        Log.d("SHIVAMMM", "copy file");

```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/58fdac3d3be16630817ec8382851f037db49d9c2/app/src/org/commcare/activities/CommCareWiFiDirectActivity.java#L815"></span>Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>